src/AppBundle/Membership/OnBoarding/RegisteringAdherent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace AppBundle\Membership\OnBoarding;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Membership\MembershipOnBoardingInterface;
+
+/**
+ * A simple instance to store a registered adherent not yet activated.
+ */
+final class RegisteringAdherent implements MembershipOnBoardingInterface
```

Comment:
Fixed in https://github.com/EnMarche/en-marche.fr/pull/221/commits/3725457429ff8f9e03b6c15570ff2718ff82401f.